README.md

```diff
@@ -57,6 +59,43 @@ check who has accessed your info with `list-access`.
 4.  run `python run_healthcheck.py` - This process will return 0 if check
     succeeded
 
+# Storage Methods
+
+If `BOT911_STORAGE_METHOD` environment variable is not set, default is `DiskStorage`, set in
+`./bot/storage/__init__.py`. Alternate value is `S3Storage` for using AWS S3.
+
+## DiskStorage
+
+### Environment Variables:
+    BOT911_STORAGE_METHOD=DiskStorage
```

Comment:
Documentation is inconsistent in how variables are presented, sometimes with `=` sometimes with `:`

Reply:
had done this to distinguish been that storage mode, where the env has to be set to the value, vs variables that must be set to user supplied values... obviously didn't work :)  .  All equals now and looks better.